src/features/userSlice.jsx

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const userSlice = createSlice({
+  name: 'user',
+  initialState: {
```

Comment:
I would create a `type` or `interface` in each `slice` called `InitialState` which has information about its initial state typing and create based on its `InitialValues` const, which would be used here for initialization